src/Uno.UI.FluentTheme.v1/Resources/Version1/PriorityDefault/ScrollBar_themeresources.xaml

```diff
@@ -197,8 +197,9 @@
         </ResourceDictionary>
     </ResourceDictionary.ThemeDictionaries>
 
-    <primitives:CornerRadiusFilterConverter x:Key="TopLeftCornerRadiusDoubleValueConverter8x" Filter="TopLeftValue" Scale="8"/>
-    <primitives:CornerRadiusFilterConverter x:Key="BottomRightCornerRadiusDoubleValueConverter8x" Filter="BottomRightValue" Scale="8"/>
+    <!-- TODO: Add Scale="8" to both lines below. They were removed because the Scale property doesn't exist in UWP builds and not implemented in WinUI builds -->
+    <primitives:CornerRadiusFilterConverter x:Key="TopLeftCornerRadiusDoubleValueConverter8x" Filter="TopLeftValue" />
+    <primitives:CornerRadiusFilterConverter x:Key="BottomRightCornerRadiusDoubleValueConverter8x" Filter="BottomRightValue" />
```

Comment:
Self reminder: Hold on this PR until https://github.com/unoplatform/uno/pull/11501 is merged and revert these changes.